functests/utils/pods/pods.go

```diff
@@ -143,3 +145,23 @@ func GetContainerIDByName(pod *corev1.Pod, containerName string) (string, error)
 	}
 	return "", fmt.Errorf("failed to find the container ID for the container %q under the pod %q", containerName, pod.Name)
 }
+
+// GetPerformanceProfilePod returns the pod running the Performance Profile Operator
+func GetPerformanceProfilePod() (*corev1.Pod, error)  {
+	selector, err := labels.Parse(fmt.Sprintf("%s=%s", "name", "performance-operator"))
+	if err != nil {
+		return nil, err
+	}
+
+	pods := &corev1.PodList{}
+
+	opts := &client.ListOptions{LabelSelector: selector, Namespace: testutils.PerformanceOperatorNamespace}
+	if err := testclient.Client.List(context.TODO(), pods, opts); err != nil {
+		return nil, err
+	}
+	if len(pods.Items) != 1 {
```

Comment:
I wonder if we should just use len(pods.Items) < 1 (just in case we eventually add HA and leader election). Or maybe it would be premature to change the test now, so just asking.

Reply:
Fair point, I prefer to keep the test as-is and see it break first when/if we add the HA/leader election.